index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div id="main">
+        <section id="navigation">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a href="index.html">Youtube</a></li>
+                </ul>
+            </nav>
+        </section>
+            
+        <section id="sidebar">
+            <div>
+                <img src="./img/thumbnail2.jpeg" alt="">
+            </div>
+            <div>
+                <img src="./img/thumbnail3.jpeg" alt="">
+            </div>
+            <div>
+                <img src="./img/thumbnail2.jpeg" alt="">
+            </div>
+            <div>
+                <img src="./img/thumbnail3.jpeg" alt="">
+            </div>
+        </section>
+    </div>
```

Comment:
The review is polite but doesn't contain the resources to find the solution or how to use the attribute `iframe`. It would be better to provide a resource that shows how to display a video on the page. Kindly consider that while providing feedback